testsuite/oauth/authorize.py

```diff
@@ -99,7 +99,12 @@ def _browser(self):
         """
         RemoteConnection.set_timeout(CONNECTION_TIMEOUT)
 
-        return webdriver.Firefox()
+        profile = webdriver.FirefoxProfile()
+        profile.set_preference('general.useragent.override', 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.116 Safari/537.36')  # noqa
```

Comment:
It looks like it works for all the other vendors, but we probably should pull it out into a config option. I'm a little sad that it's even necessary, so formalizing it into a config option feels like a painful concession, but that's still probably the way to go.